src/commands/file-actions.js

```diff
@@ -73,35 +74,40 @@ class FileActions {
       reply = `Input not recognized, please reply with ${broadcastString}(s)end, (u)ser, or (h)ash`
       state = State.FILE_TYPE
     }
-    // Make sure the file is not blank.
-    if (FileHandler.checkFileBlank(filePath)) {
-      reply = `File: ${filename} is empty. Please send a list of usernames or hashes`
-      // If file already exists go to the overwrite check state
-    } else if (fileArr.includes(`${filename}${fileAppend}`)) {
-      this.fileService.setOverwriteFileType(fileAppend)
-      reply =
-        'Warning : File already exists in user directory.\nIf you continue you will overwrite the file.\nReply (yes/no) to continue or cancel.'
-      state = State.OVERWRITE_CHECK
-      // Upload new file to the user directory
-    } else if (fileAppend === '.user' || fileAppend === '.hash') {
-      console.log('file actions user or hash, should copy file')
-      const newFilePath = `${process.cwd()}/files/${userEmail}/${filename.toString()}${fileAppend}`
-      console.log({
-        filePath,
-        newFilePath,
-        userEmail,
-        fileName: filename.toString(),
-        fileAppend,
-      })
+    if (fileAppend !== '') {
+      // Make sure the file is not blank.
+      if (FileHandler.checkFileBlank(filePath)) {
+        reply = `File: ${filename} is empty. Please send a list of usernames or hashes`
+        // If file already exists go to the overwrite check state
+      } else if (FileHandler.checkFileSize(filePath)) {
+        // TODO fix the maxentries
+        reply = `This user file has more than ${FILE_ENTRY_SIZE.value} entries. Please reduce the number of entries and try uploading it again.`
+      } else if (fileArr.includes(`${filename}${fileAppend}`)) {
+        this.fileService.setOverwriteFileType(fileAppend)
+        reply =
+          'Warning : File already exists in user directory.\nIf you continue you will overwrite the file.\nReply (yes/no) to continue or cancel.'
+        state = State.OVERWRITE_CHECK
+        // Upload new file to the user directory
+      } else if (fileAppend === '.user' || fileAppend === '.hash') {
```

Comment:
Isn't this redundant, isn't '.user' and '.hash' the only fileAppend value used?  And should this if statement replace the line 77 (if (fileAppend !== '') ?